### PR TITLE
Feature pbota 2

### PR DESCRIPTION
**Pull request by paulbota**
https://github.com/bogdibota/dashboard/issues/2

### CODE REVIEW

**Comment by paulbota** [comment]
command-status

**Comment by bogdibota** [comment]
all the components under `components/*` are dumb components. please don't link them with redux

File: src/components/statusLight/StatusLight.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
+div .container {
```
Comment (bogdibota) [inline src/components/statusLight/StatusLight.css]: refactor this with `withStyles`
animation also works: see https://github.com/mui-org/material-ui/blob/353c0aedc50de8bb2c09626e520d7cc16ac461dc/packages/material-ui/src/ButtonBase/TouchRipple.js#L84

File: src/components/statusLight/StatusLight.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+
+
+class StatusLight extends Component {
+
+  render() {
+    const {status} = this.props;
+    return (
+      <Tooltip title={ status ? status : 'UNKNOWN' } className='container'>
+        {status ?
```
Comment (bogdibota) [inline src/components/statusLight/StatusLight.js]: maybe a switch will look better here. should be easier to read

File: src/components/command/tabs/StatusTab.js [hunk 3]
```diff
@@ -1,13 +1,20 @@
...
       <div>
-        status tab
+        <StatusLight status={ status[commandId] }/> {status[commandId] }
       </div>
     );
   }
 }
 
-export default StatusTab;
+export default connect(mapStateToProps)(StatusTab);
```
Comment (bogdibota) [inline src/components/command/tabs/StatusTab.js]: see comment on MR. this should be a dumb component

File: src/components/commandTree/TreeItem.js [hunk 4]
```diff
@@ -3,30 +3,36 @@ import { withStyles } from '@material-ui/core/styles';
...
+        <TreeItemWithStyles command={ childCommand } key={ `childCommand-${ idx }` } indent={ indent + 1 } parent={ `${parent ? parent + '*' : ''}${ command.id || command.label }`}
                             onSelectCommand={ (child, idUntilNow) => onSelectCommand(child, `${ command.id || command.label }*${ idUntilNow }`) }
         />
       )),
     ]);
   }
 }
 
-const TreeItemWithStyles = withStyles(styles)(TreeItem);
+const TreeItemWithStyles = withStyles(styles)(connect(mapStateToProps)(TreeItem));
```
Comment (bogdibota) [inline src/components/commandTree/TreeItem.js]: see comment on MR. this should be a dumb component

**Review by bogdibota — CHANGES_REQUESTED** [review]

**Review by bogdibota — APPROVED** [review]

**Pull request merged**